hackerank/js-practice/hardle-race.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+//https://www.hackerrank.com/challenges/the-hurdle-race/problem
+
+function hurdleRace(power, hurdleHeight) {
+  const higherHurdle = Math.max(...hurdleHeight);
+  if (higherHurdle <= power) {
```

Comment:
Suggestion, when you have this `if-else return structure`, you can remove the else.

```
if (higherHurdle <= power) {
    return 0;
 }
 return higherHurdle - power;
 ```

What do you think? It's more readable?